project/BuildSettings.scala

```diff
@@ -54,10 +54,10 @@ object BasicSettings extends AutoPlugin {
         "-unchecked", // Enable additional warnings where generated code depends on assumptions.
         "-Xfatal-warnings", // Fail the compilation if there are any warnings.
         "-Xlint", // Enable recommended additional warnings.
-        "-Ywarn-adapted-args", // Warn if an argument list is modified to match the receiver.
+        //"-Ywarn-adapted-args", // Warn if an argument list is modified to match the receiver.
```

Comment:
Howcome these are removed ?

Reply:
They are not supported in 2.13, I did not remove them by accident.